uint32 seq
time ts
uint16 uid
uint16 hash_function_id
uint32 data_hash

